std/conv.d

```diff
@@ -1013,7 +1013,22 @@ if (!(is(S : T) &&
     !isEnumStrToStr!(S, T) && !isNullToStr!(S, T)) &&
     !isInfinite!S && isExactSomeString!T)
 {
-    static if (isExactSomeString!S && value[0].sizeof == ElementEncodingType!T.sizeof)
+    static if (is(S == struct) &&
+               __traits(hasMember, S, "toString") &&
+               isSomeFunction!(value.toString) &&
+               !__traits(isDisabled, S.toString) &&
+               is(typeof(value.toString()) == string))
+    {
+            static if (is(T == char[]))
+        {
```

Comment:
Nitpick: Please align the header of `static if` with the brace opening its body.